.gitignore

```diff
@@ -1,34 +1,27 @@
-*.py[co]
-*.sw[po]
-/pontoon/coverage.xml
-docs/_gh-pages
-build.py
-build
-dist
-.DS_Store
-node_modules
+.*
+
+*-all.js
 *-min.css
 *-min.js
-*-all.js
-.noseids
-tmp/*
-.idea
-.vscode
-*~
+*.log
 *.mo
-.vagrant
-/env/
-/static/
-/.env
-docs/_build
-.venv/
-venv/
-/media/
-.cache/
+*.py[co]
+*.sw[po]
+*~
+
+build.py
+build/
 coverage/
-*.log
+dist/
+node_modules/
+tmp/
+venv/
 
-# Docker
-.server-build
-docker/config/server.env
-docker/config/server.env-e
+/docker/config/server.env
+/docker/config/server.env-e
```

Comment:
Is that still a thing?

Reply:
Good point, I'll drop the rule. Note that you'll need to manually remove the file if you still have it; the one in my local clone was last touched in February 2022.
```suggestion
```